docs/src/manifolds/power.md

```diff
@@ -1,5 +1,56 @@
 # Power manifold
 
+A power manifold is based on a [`Manifold`](@ref) $\mathcal M$ to build a
+$\mathcal M^{n_1 \times n_2 \times \cdots \times n_m}$.
+For the two special cases $m=1$ and $m=2$ we obtain
+one-dimensional data, i.e. vectors of points on $\mathcal M$ and matrices of images,
+respectively.
```

Comment:
This sentence seems a bit hard to read, maybe it can be rephrased to something like
> In the case where $m=1$ we can represent a manifold-valued vector of data of length $n_1$, for example a time series.
> The case where $m=2$ is useful for representing manifold-valued matrices of data of size $n_1 \times n_2$, for example certain types of images.

Reply:
Yes, that was my german again peeking through. Your two sentences sound better.